### PR TITLE
Update whitelist to include trim-newlines, update lerna

### PR DESCRIPTION
**Pull request by Jkovarik**
ADR
https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=210802670

**Summary:** Summary of changes

Addresses [CUMULUS-1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

## Changes

* Updates dependencies/whitelist to address audit failures
* ADR https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=210802670

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -157,5 +157,6 @@
     "webpack-cli": "^3.3.5",
     "xml2js": "^0.4.19"
   },
-  "private": true
+  "private": true,
+  "_id": "@cumulus/core@1.0.0"
```
Comment (markdboyd) [inline package.json:161]: Why is this necessary?

**Review by markdboyd — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Jkovarik) [reply package.json:161]: 🤔  it's not.      

I'm entirely uncertain where that came from, but I'd expect it's an inadvertent add from an npm command.

Comment (Jkovarik) [reply package.json]: 32443363a
